src/cli.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import click
+from pathlib import Path
+from src.comparsion import compare_and_output
+
+def validate_paths(input_path: str, output_path: str) -> None:
+    input_path = Path(input_path)
+    
+    if not input_path.is_dir():
+        raise FileNotFoundError(f"Input path does not exist: {input_path}")
+    
+    if output_path:
+        output_path = Path(output_path)
+        if not output_path.is_dir():        
+            output_path.mkdir(parents = True, exist_ok = True)
+            print(f"Attention: new folder created at: {output_path}")
+
+    if not output_path:
+        output_path = Path(Path.cwd(), "output")
+
+        if not output_path.is_dir():        
+            output_path.mkdir(parents = True, exist_ok = True)
+            print("Default output folder created at:", output_path)
+
+    return input_path, output_path
+
+@click.command()
+@click.argument('input_path', type = click.STRING)
+@click.argument('output_path', nargs = -1, type = click.STRING)
+@click.argument('clojure_path', type = click.STRING)
```

Comment:
I'd prefer these to be options too, since it's hard to remember in which order you need to list them. Also, please add a `help` and explain that arguments mean - it's easy to assume that these are paths to input directories or files